Utils/Elasticsearch/tools/daily/es_backup

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+base_dir=/var/es
+data_dir=data
+backup_dir="$base_dir"/backup
+
+
+[ -d "$backup_dir" ] \
+  || mkdir -p -m 0700 "$backup_dir" \
+  || exit 1
+
+backup_name="$backup_dir/$data_dir"
+
+# Rotate backup files
+logrotate "$0".logrotate
```

Comment:
logrotate config file must be corresponded with variables in script.
You may check it twice while updating script or/and config or use config like template with sed-like replacements.
```
tmp_rotate_file=$(mktemp)
sed -e "s/%%%backup_name%%%/${backup_name}/" "$0".logrotate > $tmp_rotate_file
logorate $tmp_rotate_file
rm -f $tmp_rotate_file
```
and config like
```
%%%backup_name%%% {
...
}
```
or may be use directory and file name separately.

Reply:
Done: 19805770ee3ef852146801ab1448ad33d6a14f73